### PR TITLE
Pass User-Agent info to container that has the CLI in it

### PR DESCRIPTION
**Pull request by edif2008**

### CODE REVIEW

File: pkg/webhook/webhook.go [hunk 1]
```diff
@@ -384,6 +386,42 @@ func isConnectConfigurationSet(container *corev1.Container) (bool, bool) {
...
+	parts := strings.Split(strings.ReplaceAll(version, "-beta", ""), ".")
+	buildVersion := parts[0]
+	for i := 1; i < len(parts); i++ {
+		if len(parts[i]) == 1 {
+			buildVersion += "0" + parts[i]
+		} else {
+			buildVersion += parts[i]
+		}
+	}
+	fmt.Println(len(parts))
```
Comment (jillianwilson) [inline pkg/webhook/webhook.go:418]: println in not needed

**Comment by jillianwilson** [comment]
Have you tested this locally?

**Comment by edif2008** [comment]
The injector itself needs the following updates:
- Update go dependencies (may require Go 1.19)
- Change AdmissionResponse from `v1beta1` to `v1`
- Add the MetaType content to make the response complete:
  ```go
  TypeMeta: metav1.TypeMeta{
	  APIVersion: "admission.k8s.io/v1",
	  Kind:       "AdmissionReview",
  }
  ```
- Add the `vendor` directory in the repo by running `go mod vendor`
- Adjust the Dockerfile to get the proper files needed in the right way
- Add some scripts that generate and place the certificate in your Kubernetes cluster. **NOTE:** This is subject to change since we may want to do this step on the fly, when the webhook is deployed.

The working injector can be found in the [`test-env-inject`](https://github.com/1Password/kubernetes-secret-injector/tree/test-env-inject) branch.

I have tested the working version of the injector and indeed in the response it passes the User-Agent info required for the CLI.

**Review by volodymyrZotov — APPROVED** [review]

**Comment by jillianwilson** [comment]
As long as this has been tested and verified to work I am happy with this :)

**Pull request merged**